packages/runtimes/service-worker/src/ServiceWorkerRuntime.js

```diff
@@ -11,7 +11,8 @@ export default (new Runtime({
     let asset = bundle.traverse((node, _, actions) => {
       if (
         node.type === 'dependency' &&
-        node.value.specifier === '@parcel/service-worker'
+        node.value.specifier === '@parcel/service-worker' &&
+        !bundleGraph.isDependencySkipped(node.value)
```

Comment:
This was needed because there are now two such dependencies, one of the being excluded.